### ISSUE TITLE
Martingale trading in po_bot_indicators.py

### ISSUE THREAD

**Issue by KrisSzyp** [VitalySvyatyuk/pocket_option_trading_bot#11]
It works nicely, but can you provide the code that would need to be pasted to make Martingale trading work like in the po_bot.py file? Thank you in advance.
And the question is, I tried to create such a setting
if quotes[-1].close < psar[-1].sar and quotes[-2].close < supertrend[-2]:
        do_action('put')
if quotes[-1].close > psar[-1].sar and quotes[-2].close > supertrend[-2]:
        do_action('call')
but it doesn't work for me, the point is that if these two conditions are met, it should only place the transaction.
The good thing about this bot is that if you change the currency, if it drops below 92% and the Martingale, you could add it and if you don't want to use it, you set the multiplier to 1 and it wouldn't work. Also let me know if this address
BTC is ok bc1qemxzzy6rq6ycxjn0f00yqgptjqldkms8g2ucu0

Job well done.
@VitalySvyatyuk

**Comment by VitalySvyatyuk** [comment]
This btc address is good.
Your condition can be `if quotes[-1].close < psar[-1].sar and quotes[-1].close < supertrend[-1]:`
You can test with simplified condition: `quotes[-1].close < supertrend[-1]:`

**Comment by VitalySvyatyuk** [comment]
No, Martingale is not used in `po_bot_indicators`. This file is mostly for strategies testing. Just write your wise condition and test the performance. If you have profitability > 51%, then you can move such strategy to the `po_bot.py` for Martingale approach.

**Comment by KrisSzyp** [comment]
> if quotes[-1].close < psar[-1].sar and quotes[-1].close < supertrend[-1]

I get
![image](https://github.com/VitalySvyatyuk/pocket_option_trading_bot/assets/173461398/114c1dfa-779c-43a2-a317-6a0fe4d901af)
 he not get any transaction.

**Comment by KrisSzyp** [comment]
> Nie, Martingale nie jest używany w `po_bot_indicators`. Ten plik jest głównie do testowania strategii. Po prostu napisz swój mądry warunek i przetestuj wydajność. Jeśli masz rentowność > 51%, możesz przenieść taką strategię do `po_bot.py`podejścia Martingale.

**Comment by KrisSzyp** [comment]
> No, Martingale is not used in `po_bot_indicators`. This file is mostly for strategies testing. Just write your wise condition and test the performance. If you have profitability > 51%, then you can move such strategy to the `po_bot.py` for Martingale approach.

Can you show me how to transfer, for example, PSAR to Martingale, because I'm testing it and I see that with marginale I would have better results? In 7 hour i get 1 $ to transaction and i get 30 $ profit 1$ to 1 signal PSAR with assumption
![image](https://github.com/VitalySvyatyuk/pocket_option_trading_bot/assets/173461398/126e6599-2084-478f-b1bb-42cfa3c8b3d3)
The largest margin would be 4, so I would like to test it.

**Comment by VitalySvyatyuk** [comment]
you have to use supertrend[-1].upper_trend or .lower_trend as mentioned here: https://python.stockindicators.dev/indicators/SuperTrend/#content
So, for PUT use: `if supertrend[-1].upper_trend:`

**Comment by KrisSzyp** [comment]
> > if quotes[-1].close < psar[-1].sar and quotes[-1].close < supertrend[-1]
> 
> I get ![image](https://private-user-images.githubusercontent.com/173461398/347596839-114c1dfa-779c-43a2-a317-6a0fe4d901af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA2NDYwNDAsIm5iZiI6MTcyMDY0NTc0MCwicGF0aCI6Ii8xNzM0NjEzOTgvMzQ3NTk2ODM5LTExNGMxZGZhLTc3OWMtNDNhMi1hMzE3LTZhMGZlNGQ5MDFhZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxMFQyMTA5MDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NzQ5OTdhMjQ0MGEyOThiNjRhN2FjZmUwZWMyNWI1M2YxOTNmMTJhYWQ4ODQwZWM5ODE5MDJiZjc4N2NhZTI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.2sj-1n6N2AcpsTGNTsmfvM1h2GYgGgTasS6zdj9HHx0) he not get any transaction. @VitalySvyatyuk

**Comment by VitalySvyatyuk** [comment]
I moved PSAR strategy to the `po_bot.py`, and made it in `kris` branch. Use this code from here: https://github.com/VitalySvyatyuk/pocket_option_trading_bot/blob/kris/po_bot.py

**Comment by KrisSzyp** [comment]
> I moved PSAR strategy to the `po_bot.py`, and made it in `kris` branch. Use this code from here: https://github.com/VitalySvyatyuk/pocket_option_trading_bot/blob/kris/po_bot.py

Thx. 
![image](https://github.com/VitalySvyatyuk/pocket_option_trading_bot/assets/173461398/28d09cfb-f0f2-47bc-808d-59ab9aa1c8f3)

**Comment by KrisSzyp** [comment]
In your py_bot on PSAR he open a transaction when 
![image](https://github.com/VitalySvyatyuk/pocket_option_trading_bot/assets/173461398/70bcf17b-851f-4b05-bdd2-123370285b2e)
the candle will break the PSAR and not at the end of the candle, this means that the candle, for example, returns and the PSAR does not change, and there is already a transaction, it can be changed at the end of the candle.
